### ISSUE TITLE
10 - SCREEN READER

### ISSUE THREAD

**Issue by prodb** [pace-running/pace3#12]
**As an** event attendee
when I am on any page of the website and have visual impairments
**I want** that the info to be read aloud to me
**So that** I can understand the information on the website.

Business Context:
In order to enable users with visual impairments (e.g. with dyslexia) to perceive the information effectively, a screen reader function should be presented clearly on every page that reads the content aloud to the user.

Acceptance Criteria:
**Given** that I am on any page
**When** I click on the screen reader button
**Then** the information on the current page is read aloud from top to bottom.
